src/app/pages/test-results-search/components/advanced-search/advanced-search.ts

```diff
@@ -91,13 +93,18 @@ export class AdvancedSearchComponent {
   ) {
   }
 
-  upperCaseAlphaNum(event: any): void {
-    if (typeof event.target.value !== 'string') return;
+  upperCaseAlphaNum(event: InputChangeEventDetail | InputInputEventDetail, field: string): void {
+    if (typeof event.value !== 'string') return;
 
-    if (nonAlphaNumericValues.test(event.target.value)) {
-      event.target.value = event.target.value?.replace(nonAlphaNumericValues, '').toUpperCase();
+    // Added logic here as it is used on the (ionInput) attribute of the staffNo. input field, sets toggle to unchecked
```

Comment:
as soon as its merged this comment will have zero meaning as it only refers to code that no longer exists

Reply:
I think this annotation will make sense and refer to this line no?
`if (event.value === '' && (field === 'staffId')) {`

Reply:
yes that comment refers to the line below not the one you thought andy

Reply:
misunderstood, retracted